infrastructure/config/config_yaml.py

```diff
@@ -0,0 +1,42 @@
+import yaml
+
+from infrastructure.config.thresholds import Thresholds
+
+
+class ConfigYAML:
+    """ ConfigYAML implements domain.config.config.Config protocol """
+
+    @property
+    def test_id(self) -> str:
+        return self._test_id
+
+    @property
+    def data_update_period_seconds(self) -> int:
+        return self._data_update_seconds
+
+    @property
+    def data_update_lookback_seconds(self) -> int:
+        return self._data_update_lookback_seconds
+
+    @property
+    def latency(self) -> Thresholds:
+        return self._latency
+
+    @property
+    def jitter(self) -> Thresholds:
+        return self._jitter
+
+    @property
+    def packet_loss(self) -> Thresholds:
+        return self._packet_loss
+
+    def __init__(self, filename: str) -> None:
+        with open(filename, "r") as file:
+            config = yaml.load(file, yaml.FullLoader)
```

Comment:
We can really use `SafeLoader` (`config = yaml.safe_load(file)`), because we will never have to construct any complex objects or use references and overrides.